README.md

```diff
@@ -74,6 +75,8 @@ fuq.in({ kabooM: '!' }); // Error: Missing `code`, ...
 
 ```
 
+## Debugging
+Pubsub implementation allows debug mode with debug logs, you need to set `NODE_DEBUG` variable to `fuqu_pubsub`.
```

Comment:
Why to use an adapter's name in a global readme for logging?

I aggree it might be useful in some cases to have the ability to enable logging for different adapters, but then it should be in adapter options.

In a global scope i would rather see simple `fuqu` for enabling logging for the entire lib.